lcls_tools/common/measurements/wire_scan.py

```diff
@@ -166,6 +133,15 @@ def scan_with_wire(self):
         and allows time for the buffer to update its state.
         """
         # Start wire scan
+        if self.my_buffer is None:
```

Comment:
Is this code repeated from line 49+? Maybe it can also be pulled out if it's general enough to re-use?